pkg/kubelet/cm/node_container_manager_linux.go

```diff
@@ -190,6 +191,21 @@ func (cm *containerManagerImpl) getNodeAllocatableAbsoluteImpl(capacity v1.Resou
 		}
 		result[k] = value
 	}
+
+	// for every huge page reservation, we need to remove it from allocatable memory
+	for k, v := range result {
```

Comment:
```suggestion
	for k, v := range capacity {
```

Reply:
I believe this should be calculated by capacity not allocatable?

Reply:
since memory uses does not include hugepage uses，here we should use  allocatable

Reply:
> since memory uses does not include hugepage uses，here we should use allocatable

Not sure if I follow. If we reserve huge page memory in system-reserved and/or kube-reserved, we would like to decrement by all huge page memory, not only allocatable. Or?